test/conformance/kernel/urKernelRetain.cpp

```diff
@@ -17,3 +18,27 @@ TEST_P(urKernelRetainTest, InvalidNullHandleKernel) {
     ASSERT_EQ_RESULT(UR_RESULT_ERROR_INVALID_NULL_HANDLE,
                      urKernelRetain(nullptr));
 }
+
+TEST_P(urKernelRetainTest, CheckReferenceCount) {
+    uint32_t referenceCount = 0;
+    ASSERT_SUCCESS(urKernelGetInfo(kernel, UR_KERNEL_INFO_REFERENCE_COUNT,
+                                   sizeof(referenceCount), &referenceCount,
+                                   nullptr));
+    ASSERT_EQ(referenceCount, 1);
+
+    {
+        ASSERT_SUCCESS(urKernelRetain(kernel));
+    }
```

Comment:
This scope seems unnecessary, it's also something `clang-format` wants to [change](https://github.com/oneapi-src/unified-runtime/actions/runs/12260313456/job/34204620809?pr=2444).

Reply:
Must've fat fingered that! It's gone now, thanks.